scripter/scripter.go

```diff
@@ -154,3 +167,75 @@ func SetBasicMethods(c ScrConn, service string) {
 		}
 	}, service)
 }
+
+// setScriptInterval sets the interval of checking whether scripts have been changed
+func SetScriptInterval(s Scripter) {
```

Comment:
This function does not set the interval at all. It merely enables checking the scripts. If it was going to set the interval, maybe add it as a parameter?